src/applications/static-pages/static-page-widgets.js

```diff
@@ -6,7 +6,7 @@ function mountWidgets(widgets) {
       var slowLoadingThreshold = (widget.slowLoadingThreshold || 6) * 1000;
       var slowMessage = widget.slowMessage || 'Sorry, this is taking longer than expected.';
 
-      if (!widget.showSpinnerUnauthed && sessionStorage.userToken) {
+      if (!widget.showSpinnerUnauthed) {
```

Comment:
I don't think the sessionStorage check here made a difference... I still saw a blink of the loading indicator when I went to the relevant pages in Prod. Probably safe to remove here.

@cehsu: I noticed you added the application status widget to some of the static pages, so just letting you know in case that wasn't expected behavior.